tests/functional/spec/features/calculated_summary_repeating_section.spec.js

```diff
@@ -1,33 +1,33 @@
-import SubmitPage from "../../generated_pages/new_calculated_summary_repeating_section/personal-details-section-summary.page.js";
+import SetMinMaxBlockPage from "../../generated_pages/calculated_summary_repeating_section/set-min-max-block.page.js";
+import CurrencyTotalPlaybackPageSkippedFourth from "../../generated_pages/calculated_summary_repeating_section/currency-total-playback-skipped-fourth.page.js";
+import UnitTotalPlaybackPage from "../../generated_pages/calculated_summary_repeating_section/unit-total-playback.page.js";
+import PercentageTotalPlaybackPage from "../../generated_pages/calculated_summary_repeating_section/percentage-total-playback.page.js";
+import NumberTotalPlaybackPage from "../../generated_pages/calculated_summary_repeating_section/number-total-playback.page.js";
+import BreakdownPage from "../../generated_pages/calculated_summary_repeating_section/breakdown.page.js";
+import SecondCurrencyTotalPlaybackPage from "../../generated_pages/calculated_summary_repeating_section/second-currency-total-playback.page.js";
+import CalculatedSummaryTotalConfirmation from "../../generated_pages/calculated_summary_repeating_section/calculated-summary-total-confirmation.page.js";
+import SubmitPage from "../../generated_pages/calculated_summary_repeating_section/personal-details-section-summary.page.js";
 import ThankYouPage from "../../base_pages/thank-you.page.js";
 import HubPage from "../../base_pages/hub.page.js";
-import PrimaryPersonListCollectorPage from "../../generated_pages/new_calculated_summary_repeating_section/primary-person-list-collector.page";
-import PrimaryPersonListCollectorAddPage from "../../generated_pages/new_calculated_summary_repeating_section/primary-person-list-collector-add.page.js";
-import ListCollectorPage from "../../generated_pages/new_calculated_summary_repeating_section/list-collector.page";
-import ListCollectorAddPage from "../../generated_pages/new_calculated_summary_repeating_section/list-collector-add.page";
+import PrimaryPersonListCollectorPage from "../../generated_pages/calculated_summary_repeating_section/primary-person-list-collector.page";
+import PrimaryPersonListCollectorAddPage from "../../generated_pages/calculated_summary_repeating_section/primary-person-list-collector-add.page.js";
+import ListCollectorPage from "../../generated_pages/calculated_summary_repeating_section/list-collector.page";
+import ListCollectorAddPage from "../../generated_pages/calculated_summary_repeating_section/list-collector-add.page";
 
 describe("Feature: Calculated Summary Repeating Section", () => {
   describe("Given I have a Calculated Summary in a Repeating Section", () => {
     before("Get to Calculated Summary", () => {
-      browser.openQuestionnaire("test_new_calculated_summary_repeating_section.json");
+      browser.openQuestionnaire("test_calculated_summary_repeating_section.json");
```

Comment:
As above, the same applies for the repeats

Reply:
Reverted this test file after bringing back the schema